dmutils/content_loader.py

```diff
@@ -175,6 +175,9 @@ def get_data(self, form_data):
         in the form data are ignored. Fields in the form data are parsed according
         to their type in the section data.
         """
+        # strip trailing and leading whitespace from form values
+        form_data = ImmutableMultiDict(_strip_values(form_data.to_dict(flat=False)))
```

Comment:
I think using `form_data.iteritems(multi=True)` here might simplify `_strip_values` a bit. My first though would be that:

    form_data = ImmutableMultiDict((k,v.strip()) for k,v in form_data.iteritems(multi=True))

should be enough, but I'd test this with assertions, features and benefits, pricing, and file uploads to check that it covers the edge cases.